config/environments/development.rb

```diff
@@ -43,5 +43,6 @@
   config.action_mailer.default_url_options = {
     host: 'localhost:3000'
   }
+  config.action_controller.default_url_options = { host: 'www.example.com' }
```

Comment:
Hi, this is a great fix -- ought this to be `localhost:3000` to match the default on line 44? 

Thanks!! 😄

Reply:
@jywarren Yes you are right! I will do it right now! :)

Reply:
Done!